plugins/modules/elb_application_lb.py

```diff
@@ -514,9 +525,12 @@ def create_or_update_elb(elb_obj):
 
     # Listeners
     listeners_obj = ELBListeners(elb_obj.connection, elb_obj.module, elb_obj.elb['LoadBalancerArn'])
-
     listeners_to_add, listeners_to_modify, listeners_to_delete = listeners_obj.compare_listeners()
 
+    # Exit on check_mode
+    if elb_obj.module.check_mode and (listeners_to_add or listeners_to_modify or listeners_to_delete):
+        elb_obj.module.exit_json(changed=True, msg='Would have updated ELB if not in check mode.')
```

Comment:
@alinabuzachis I stole that idea from [ec2_vol](https://github.com/ansible-collections/amazon.aws/pull/509/files), but I actually really like the idea of having a message there! Is there a standard way of returning on check_mode?

Reply:
> Is there a standard way of returning on check_mode?

Nope!  :smile: 
In the majority of cases AWS modules will do one of:
* Return a `msg` like you have here
* Return only a `changed` key (based on if there would have been a change, if not in check mode)
* Return something that resembles the normal RETURN of the module (like Alina has noted)

Some of this is just the legacy of 7 or 8 years of organic changes to the AWS modules.  Some of it is what makes the most sense for the way the module code has been structured (since we also don't have a defined standard that we expect for that either).  The Ansible devel guide doesn't dictate a preference for what check_mode should return, and the user guide doesn't promise anything, so it's collection/developer preference.  I'm personally ok with having check_mode return only a `changed` or having a `msg`.